### DISCUSSION TITLE
Variables in Policies or Powershell Scripts

### DISCUSSION THREAD

**Discussion by jleroxGWNTC** [KelvinTegelaar/CIPP#1818] [General]
I have a peak through the documentation, but I wanted to see if where possible to have variables or calls relative to the client in policies or scripts. For example, we want to deploy a licensing@client.tld shared mailbox that then forwards to an internal mailbox which is organised by client code.  Client ABC would be clients+abc@msp.tld (now this isn't overly important just helps sorts items when they arrive)

So we need to create a shared mailbox, super easy in powershell, then we need to have a outbound filter policy that allows licensing@client.tld to forward to clients@msp.tld or preferably clients+ABC@msp.tld.  I know we can't manage Outbound Spam Policies in CIPP, but to create the policy and then place it with powershell, I want to pass on @client.tld and client. Or is there a way to manage this or run the script from CIPP?

Would it be best to pull the default client domain and do if =clienta.onmicrosoft.com then $CLIENTCODE = ABC else if; 
obviously in a better structure than that.

**Comment by KelvinTegelaar** [comment]
Big ask and more of a feature request - We're working on something for this in the background but currently don't have it yet.
